docs/index.md

```diff
@@ -0,0 +1,562 @@
+---
+layout: page-fullwidth
+order: 1
+title: "Hand Coded Heat"
+subheadline: "Hello World for Numerical Packages"
+teaser: "Why use numerical packages..."
+permalink: "lessons/hand_coded_heat/"
```

Comment:
Whoops...this line means that the GH pages generation will wind up putting this page at 

`https://josee1031.github.io/hello-numerical-world` + `lessons/hand_coded_heat/` which means the link I put in `README.md` needs to be adjusted to that.